packages/framework/src/Twig/UploadedFileExtension.php

```diff
@@ -107,16 +110,12 @@ public function getUploadedFilePreviewHtml(UploadedFile $uploadedFile): string
      */
     protected function getUploadedFileIconHtml(string $uploadedFileIconType): string
     {
-        $classes = [
-            'svg',
-            'svg-file-' . $uploadedFileIconType,
-            'list-files__item__file__type',
-            'list-files__item__file__type--' . $uploadedFileIconType,
-            'text-no-decoration',
-            'cursor-pointer',
-        ];
-
-        return '<i class="' . implode(' ', $classes) . '"></i>';
+        return
+            '<span class="list-files__item__file__type list-files__item__file__type--' .
+            $uploadedFileIconType .
+            ' text-no-decoration cursor-pointer">' .
+            $this->iconRenderer->renderIcon('tabler:file-type-' . $uploadedFileIconType) .
```

Comment:
This will not lock icons. Instead of that you will always get them on demand

Reply:
:thinking: so how to configure it to lock the icons?